themes/default/layouts/partials/registry/package/right-nav.html

```diff
@@ -21,8 +21,18 @@ <h4 class="no-anchor">
                 </div>
             {{ end }}
         </div>
+        <div class="pl-3 flex-1 overflow-y-auto border-b-2 pb-3 pt-2">
+            <h4 class="no-anchor mt-2 font-semibold font-body text-sm" style="color: #484848;">
+                <a href="#" class="contents-heading">Pulumi AI</a>
+            </h4>
+            <section class="text-sm">
+                <p class="m-2">Need help using this package?</p>
+                <p class="m-2">Try out <i class="fas fa-robot mr-1 text-xs relative text-gray-800" style="top: -1px;"></i><a target="_blank" href="/ai/" class="underline">Pulumi AI</a>!</p>
```

Comment:
I would love to contextualize these links even more.

When on https://www.pulumi.com/registry/packages/aws/api-docs/ec2/instance/, I'd love to send the user to `/ai` with a query string setting the prompt to "Create an `aws.ece.Instance` resource" or similar.

I'd also love to not say "using this package", but to contextualize this to the individual resource.

For pages that aren't resource pages, we can back off to "use this package" or "use this module" with a similar approach.

Relatedly, I might phrase as "Explore `aws.ec2.Instance` with Pulumi AI" instead of "Try out Pulumi AI".

Reply:
My very poor attempt at this from Friday :-). 

```html
<img width=20 src="/registry/packages/einsteinypus.svg" style="display: inline-block"/> Explore <a href="/ai?prompt=Create%20an%20%60{{ $.Page.Params.title_tag }}%60%20resource">{{ $.Page.Params.title_tag }}</a>
```